src/app/core/config-ui/config-entity/config-entity.component.spec.ts

```diff
@@ -0,0 +1,169 @@
+import {
+  ComponentFixture,
+  fakeAsync,
+  TestBed,
+  tick,
+} from "@angular/core/testing";
+
+import { ConfigEntityComponent } from "./config-entity.component";
+import { ConfigService } from "../../config/config.service";
+import { EntityMapperService } from "../../entity/entity-mapper/entity-mapper.service";
+import { mockEntityMapper } from "../../entity/entity-mapper/mock-entity-mapper-service";
+import { Config } from "../../config/config";
+import { EntityActionsService } from "../../entity/entity-actions/entity-actions.service";
+import { CoreModule } from "../../core.module";
+import { CoreTestingModule } from "../../../utils/core-testing.module";
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+import { EntityTypeLabelPipe } from "../../common-components/entity-type-label/entity-type-label.pipe";
+import {
+  EntityDetailsConfig,
+  Panel,
+} from "../../entity-details/EntityDetailsConfig";
+import { Entity } from "../../entity/model/entity";
+import { MatTabsModule } from "@angular/material/tabs";
+import { NoopAnimationsModule } from "@angular/platform-browser/animations";
+import { EntitySchemaField } from "../../entity/schema/entity-schema-field";
+import { DatabaseEntity } from "../../entity/database-entity.decorator";
+import { DatabaseField } from "../../entity/database-field.decorator";
+
+describe("ConfigEntityComponent", () => {
+  let component: ConfigEntityComponent;
+  let fixture: ComponentFixture<ConfigEntityComponent>;
+
+  let mockConfigService: jasmine.SpyObj<ConfigService>;
+
+  let config;
+  let viewConfig: EntityDetailsConfig;
+  let viewConfigId, entityConfigId;
+
+  @DatabaseEntity("AdminTest")
+  class AdminTestEntity extends Entity {
+    static readonly ENTITY_TYPE = "AdminTest";
+
+    @DatabaseField({ label: "Name" }) name: string;
+  }
+
+  beforeEach(() => {
+    viewConfigId = `view:${AdminTestEntity.route.substring(1)}/:id`;
+    entityConfigId = `entity:${AdminTestEntity.ENTITY_TYPE}`;
+    viewConfig = {
+      entity: AdminTestEntity.ENTITY_TYPE,
+      panels: [{ title: "Tab 1", components: [] }],
+    };
+    config = {
+      [viewConfigId]: { component: "EntityDetails", config: viewConfig },
+      [entityConfigId]: {},
+    };
+
+    mockConfigService = jasmine.createSpyObj(["getConfig"]);
+    mockConfigService.getConfig.and.returnValue(config[viewConfigId]);
+
+    TestBed.configureTestingModule({
+      declarations: [ConfigEntityComponent],
+      schemas: [NO_ERRORS_SCHEMA], // keep this a unit test without its child component
```

Comment:
I came fairly close to loosing my mind over config not being loaded correctly. but I guess now it works anyway since I turned this into a standalone component ...